highfidelity/hifi
# FB16992 - Layout changes to the two lightboxes in the Avatar app

https://highfidelity.fogbugz.com/f/cases/16992/Layout-changes-to-the-two-lightboxes-in-the-Avatar-app

**Test plan**

1. Open avatarapp
2. Click 'get more avatars' (at the end of bookmarks)
3. Ensure popup has the same data/layout as on the image below (from https://highfidelity.fogbugz.com/f/cases/16992/Layout-changes-to-the-two-lightboxes-in-the-Avatar-app): 

![image](https://user-images.githubusercontent.com/23638/43100990-bdf3d756-8ecf-11e8-9ac8-1e81ed4338ca.png)

4. Click 'Add' button to show 'get wearables' popup (button visible if bookmark with no wearables selected)
5. Ensure popup has the same data/layout as on the image below (https://highfidelity.fogbugz.com/f/cases/16992/Layout-changes-to-the-two-lightboxes-in-the-Avatar-app): 

![image](https://user-images.githubusercontent.com/23638/43101109-18bcfdf2-8ed0-11e8-8ea4-66b3ed0c386b.png)
